.github/workflows/osrm-backend.yml

```diff
@@ -506,6 +506,11 @@ jobs:
     - name: Install dev dependencies
       run: |
         python3 -m pip install conan==1.53.0
+        # workaround for issue that GitHub Actions seems to not adding it to PATH after https://github.com/actions/runner-images/pull/6499
+        # and that's CI cannot find conan executable installed above
+        if [[ "${RUNNER_OS}" == "macOS" ]]; then
```

Comment:
I should have created an issue in https://github.com/actions/runner-images repo and I bet there is better fix in general(at least we could detect `/Library/Frameworks/Python.framework/Versions/3.11` string automatically), but just want to unblock @DennisOSRM with his PRs